### DISCUSSION TITLE
v2.1.1

### DISCUSSION THREAD

**Discussion by clementwzk** [clementwzk/OpenCalc#165] [Announcements]
**New:**
- Explicit errors (@lefterisgar)
- Redesign of the history
- Add timestamps to history results (@lefterisgar)
- Do not allow duplicate math symbols on input (@lefterisgar)

**Updated:**
- arccos, arcsin and arctan are now displayed as cos⁻¹, sin⁻¹ and tan⁻¹

<hr /><em>This discussion was created from the release <a href='https://github.com/Darkempire78/OpenCalc/releases/tag/v2.1.0'>v2.1.0</a>.</em>

**Comment by lefterisgar** [comment]
The latest release (v2.1.1) wasn't minified for some reason. We might need to further investigate. Did you select "debug" while building it?

**Reply by clementwzk** [reply]
It's normal, #164 was merged after the v2.1.0

**Reply by lefterisgar** [reply]
I'm referring to **v2.1.1** whose size is bigger than the previous release (v2.0.0)!

**Reply by clementwzk** [reply]
I just built an apk and it takes 2mo

**Reply by lefterisgar** [reply]
Really? I'm very happy to hear that!
